src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.2
++++++
+* Added new commands for budgets, marketplace and pricesheet APIs
```

Comment:
This should be reworded because it is no longer true. I would recommend something like:
```
* Added `consumption pricesheet show` command.
* `consumption usage list`: Added support to list by billing period.
...
```